README.md

```diff
@@ -15,3 +16,60 @@
 
 ### Run unit tests
     yarn test:unit
+
+## Using this library
+
+Package requirements, to be added after `vue create ...`
+
+```bash
+# Vuetify
+vue add vuetify
+
+# Vee Validate
+yarn add vee-validate
+
+# FontAwesome
+yarn add @fortawesome/fontawesome-free -D
+# OR add the CDN link in index.html
+# <link href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" rel="stylesheet">
+```
+
+To use components in your Vue.js app, the Vue instance must meet some requirements.  The basic requirements are:
+
+```javascript
+Vue.use(Vuetify, {
+  iconfont: 'fa',
+});
+Vue.use(VeeValidate);
+
+new Vue({
+  render: h => h(App),
+  provide: { girderRest },
+}).$mount('#app');
+```
+
+Because these are vuetify components, your consumer application is responsible for creating the `v-app` container.  [Read more](https://vuetifyjs.com/en/layout/pre-defined#all-about-app).  See `src/main.js` for a more comprehensive example.
+
+### Components
+
+Public components live in `src/components` and should be imported by name.
+
+* Upload
+* Authentication
+
+### Event API
```

Comment:
#11 is now merged to master and provides `login`, `logout`, and `register` events on the `girderRest` injection. So let's use that to avoid introducing this new bus.

Reply:
Let's remove this whole section for now, it's probably premature to impose this as a documentation burden, and it's currently only documenting low-level events that are probably only useful to developers of this library. IMO the documentation for downstream-facing events should probably be auto-generated if possible.